pyProbeParticle/HighLevel.py

```diff
@@ -212,7 +216,7 @@ def computeELFF_pointCharge( geomFile, tip='s', save_format=None, computeVpot=Fa
     if(verbose>0): print("<<<END: computeELFF_pointCharge()")
     return FF, V, nDim, lvec
 
-def computeElFF(V,lvec,nDim,tip,computeVpot=False, tilt=0.0 ):
+def computeElFF(V,lvec,nDim,tip,computeVpot=False, tilt=0.0,sigma=PPU.params['sigma'] ):
```

Comment:
This is probably not doing what you think it does. The function signature gets evaluated only once when the module is imported, so the `sigma` value here will not get updated even if the value in `PPU.params['sigma']` changes. It's probably better if you set the default value to `None` and then inside function check if it's `None` and set it to `PPU.params['sigma']` if it is.

Reply:
Hi Nico,
Thanks for the review. I rewrote the function acording to your comments and tested it againg. I think it is correct now.

Reply:
Ok, looks good.